eng/pipelines/pr.yml

```diff
@@ -0,0 +1,12 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+      - feature/track2
```

Comment:
We should probably go ahead and add the other normal branch patterns here as well. They shouldn't trigger on main until this is in main so I think it should be safe.

Reply:
Added as:
```yaml
pr:
  branches:
    include:
      - main
      - feature/*
      - hotfix/*
      - release/*
```